src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/InputAbstractClassNameFormerFalsePositive.java

```diff
@@ -1,12 +1,27 @@
 package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
+/* Config:
+ * default
+ *
+ */
```

Comment:
it is a minor request but could you reformat inputs in the following way

```
package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;

/*
 * Config: default
 */
abstract public class Inpu....
```

Reply:
Yeah I will